components/intake.py

```diff
@@ -12,10 +11,12 @@ class Intake:
     extension_arm_right: wpilib.Solenoid
     infrared: wpilib.AnalogInput
     cube_switch: wpilib.DigitalInput
+    joystick: wpilib.Joystick
 
     def setup(self):
         """This is called after variables are injected by magicbot."""
-        pass
+        self.intake_right.set(self.intake_right.ControlMode.Follower, 2)
```

Comment:
What on earth is going on here. Even the tests don't know what ID 2 is.

Use [TalonSRX.follow()](https://robotpy.readthedocs.io/projects/ctre/en/latest/api_talonsrx.html#ctre.talonsrx.TalonSRX.follow), and pass in an actual TalonSRX object.